src/Sentry/Sentry.csproj

```
@@ -5,6 +5,8 @@
     <NoWarn Condition="'$(TargetFramework)' == 'netstandard2.0'">$(NoWarn);RS0017</NoWarn>
     <CLSCompliant Condition="'$(TargetPlatformIdentifier)' == ''">true</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsTrimmable>true</IsTrimmable>
+    <DefineConstants>$(DefineConstants);IsTrimmable</DefineConstants>
```

The convention is all caps, how about we go with:

```suggestion
    <DefineConstants>$(DefineConstants);TRIMMABLE</DefineConstants>
```